src/librustdoc/html/render/mod.rs

```diff
@@ -3048,6 +3048,7 @@ fn item_struct(w: &mut Buffer, cx: &Context, it: &clean::Item, s: &clean::Struct
                     ItemType::StructField,
                     field.name.as_ref().unwrap()
                 ));
+                debug!("LOOKIE: ty={:?}, printed={}", ty, ty.print());
```

Comment:
:laughing: I think this might be better with a different prefix
```suggestion
                debug!("item_struct: ty={:?}, printed={}", ty, ty.print());
```

Reply:
Sorry, I thought I removed that debugging code! I use `LOOKIE` so it’s easy to search for in the logs :)

Reply:
I ended up just removing the debugging code since I didn't think it was useful enough. It's fairly straightforward to re-add if needed in the future.